dev/com.ibm.ws.org.apache.cxf.cxf.rt.rs.client.3.2/src/org/apache/cxf/jaxrs/client/AbstractClient.java

```diff
@@ -375,6 +375,10 @@ public void close() {
         closed = null;
     }
 
+    public void removeAllHeaders() {
```

Comment:
This looks to be a duplicate of removeAllHeaders() on [L175](https://github.com/andymc12/open-liberty/blob/f4a9461693694e01b31feb3c3ad4adf5e5e6b894/dev/com.ibm.ws.org.apache.cxf.cxf.rt.rs.client.3.2/src/org/apache/cxf/jaxrs/client/AbstractClient.java#L175).